core/src/test/java/org/infinispan/persistence/file/SingleFileStorePopulator.java

```diff
@@ -0,0 +1,70 @@
+//package org.infinispan.persistence.file;
+//
+//import java.nio.file.Path;
+//import java.nio.file.Paths;
+//import java.util.stream.IntStream;
+//
+//import org.infinispan.Cache;
+//import org.infinispan.configuration.parsing.ConfigurationBuilderHolder;
+//import org.infinispan.manager.DefaultCacheManager;
+//import org.infinispan.manager.EmbeddedCacheManager;
+//import org.infinispan.persistence.spi.MarshallableEntryFactory;
+//import org.infinispan.test.TestingUtil;
+//
+///**
+// * // TODO: Document this
+// *
+// * @author Ryan Emerson
+// * @since 13.0
+// */
+//public class SingleFileStorePopulator {
```

Comment:
maybe move this class to tools module?

Reply:
I've removed the file. The content of it is in the `SingleFileGracefulShutdownMigrationTest` JavaDocs.